docs/design/autoware-architecture/vehicle/index.md

```diff
@@ -1,5 +1,117 @@
-# Vehicle component design
+# Vehicle interface design
 
-!!! warning
+## Abstract
 
-    Under Construction
+The DBW component provides an interface between Autoware and a vehicle that passes control signals to the vehicle’s drive-by-wire system and receives vehicle information that is passed back to Autoware.
+
+## 1. Requirements
+
+Goals:
+
+- The DBW component converts Autoware commands to a vehicle-specific format and converts vehicle status in a vehicle-specific format to Autoware messages.
+- The interface between Autoware and the DBW component is abstracted and independent of hardware.
+- The interface is extensible such that additional vehicle-specific commands can be easily added. For example, headlight control.
+
+Non-goals:
+
+- Accuracy of responses from the vehicle will not be defined, but example accuracy requirements from reference designs are provided as examples.
+- Response speed will not be defined.
+
+## 2. Architecture
+
+The DBW component consists of the following components:
+
+- A Raw Vehicle Command Converter component that will pass through vehicle commands from the Control component if velocity/acceleration control is supported by the drive-by-wire system. Otherwise, the Control commands will be modified according to the control method (eg: converting a target acceleration from the Control component to a vehicle specific accel/brake pedal value through the use of an acceleration map)
+- A Vehicle Interface component (vehicle specific) that acts as an interface between Autoware and a vehicle to communicate control signals and to obtain information about the vehicle (steer output, tyre angle etc)
+
+Each component contains static nodes of Autoware, while each module can be dynamically loaded and unloaded (corresponding to C++ classes). The mechanism of the DBW component is depicted by the following figures:
+
+<!-- insert vehicle component architecture diagram here
+![alt_text](images/image1.png "image_tooltip")
+-->
+
+<!-- insert vehicle component overview diagram here
+![alt_text](images/image1.png "image_tooltip")
+-->
+
+## 3. Features
```

Comment:
Currently `Features` include:
- Diagnostics
- Safety features
- Optional controls

we probably should include basic function as well:
- Converting Autoware control commands to vehicle specific command
- Converting vehicle specific status information (velocity, steering) to Autoware status message

Although we already explain them in the "Requirements" section, I think we should mention it again.